api/v2alpha1/python/kfp_pipeline_spec/__init__.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2020 Google LLC
```

Comment:
Can we ensure the namespace is `kfp.pipeline_spec` instead of `kfp_pipeline_spec` ?

Reply:
Updated.